claim.schema.json

```diff
@@ -14,51 +14,52 @@
           "type": "object",
           "properties": {
             "tnf": {
               "type": "string",
-              "description": "The test-network-function (tnf) version."
-            },
-            "openshift": {
-              "type": "string",
-              "description": "The test-network-function (tnf) version."
+              "description": "The test-network-function (tnf) release version."
             }
           },
           "additionalProperties": false,
-          "required": ["openshift", "tnf"]
+          "required": [
+            "tnf"
+          ]
         },
         "configurations": {
           "type": "object",
-          "description": "Key/Value where the key is the configuration name.",
+          "description": "Tests within test-network-function often require configuration.  For example, the generic test suite requires listing all CNF containers.  This information is used to derive per-container IP address information, which is then used as input to the connectivity test suite.  Test suites within test-network-function may use multiple configurations, but each with a unique name.",
           "additionalProperties": {
             "type": "object",
-            "description": "Key/Value where the key is the configuration item."
+            "description": "Tests within test-network-function often require configuration.  For example, the generic test suite requires listing all CNF containers.  This information is used to derive per-container IP address information, which is then used as input to the connectivity test suite.  Test suites within test-network-function may use multiple configurations, each of which is arbitrary in structure and use case specific."
           }
         },
-        "hosts": {
+        "nodes": {
           "type": "object",
-          "description": "Hostname/payload key/value information for all hosts in the cluster.",
+          "description": "An OpenShift cluster is composed of an arbitrary number of Nodes used for platform and application services.  Since a claim must be reproducible, a variety of per-Node information must be collected and stored in the claim.  Node names are unique within a given OpenShift cluster.",
           "additionalProperties": {
             "type": "object",
-            "description": "A description of a host using the Unix lshw command."
+            "description": "An OpenShift cluster is composed of an arbitrary number of Nodes used for platform and application services.  Since a claim must be reproducible, a variety of per-Node information must be collected and stored in the claim.  The payload for a given Node is arbitrary, since the information gathered for a particular Node depends on its Roles."
           }
         },
         "results": {
           "type": "object",
-          "description": "The test results."
+          "description": "The test-network-function test results.  Results are formatted as JSON JUnit output."
```

Comment:
"as JSON JUnit output"

Do you mean there is a JSON JUnit output format and the data is consistent with it? Or that this is "a JSON representation from JUnit output"? Or something else?

Reply:
The latter.  I will update the description to reflect that.